lib/format.js

```diff
@@ -22,7 +22,7 @@ function selectSolutions(editor, message) {
 	}
 
 	const position = new Range(...range.map(index => buffer.positionForCharacterIndex(index)));
-	return {position, replaceWith: text};
+	return [{position, replaceWith: text}];
```

Comment:
Solutions on lint messages should be an array

See https://github.com/steelbrain/linter/blob/master/docs/types/linter-message-v2.md